app/script/view_model/content/TitleBarViewModel.js

```diff
@@ -80,9 +80,8 @@ z.viewModel.content.TitleBarViewModel = class TitleBarViewModel {
     });
 
     this.supportsVideoCall = ko.pureComputed(() => {
-      return this.showCallControls()
-        ? this.userRepository.isTeam() && this.conversationEntity().supportsVideoCall(true)
-        : false;
+      const userAllowedVideoCall = this.userRepository.isTeam() || this.conversationEntity().is_one2one();
+      return this.showCallControls() && this.conversationEntity().supportsVideoCall(true) && userAllowedVideoCall;
```

Comment:
This condition doesn't seem to be semantically correct `this.showCallControls`.
From a reader point of view, I don't really understand how the fact that supporting video call is related to the fact that we want to show the call controls.